packages/node_modules/@webex/internal-plugin-calendar/src/util.js

```diff
@@ -0,0 +1,10 @@
+const CalendarUtil = {
+
+  derive(item) {
+    item.endTime = new Date(new Date(item.start).getTime() + item.durationMinutes * 60000);
+
+    return item;
```

Comment:
Some code comments would go a long way here. I don't know the purpose/intent of this.

Reply:
will add